database/migrations/2018_12_21_175437_populate_skills.php

```diff
@@ -63,7 +63,7 @@ public function up()
             ['skill_type_id' => 2, 'name' => 'project_management'],
             ['skill_type_id' => 2, 'name' => 'stakeholder_relations'],
             ['skill_type_id' => 2, 'name' => 'dot_net'],
-            ['skill_type_id' => 2, 'name' => 'geospacial_programming'],
+            ['skill_type_id' => 2, 'name' => 'geospatial_programming'],
```

Comment:
You shouldn't ever go back to an old migration and make changes, but instead create a new to make whatever changes you need. However, I think this case might be fine since it's only fixing a typo on the name field and not manipulating the structure or a value that is referenced anywhere. Let me double check and get back to you though.

Reply:
So after talking to Tristan, you'll need to undo this change and instead create a new migration that fixes the spelling. This is due to the fact that it won't actually change the value in production, but make it seem like it did. Plus, you also fixed the spelling in the lang files which are getting referenced by the DB, so this will break the connnection.

Reply:
Yep, I recognized the fact that I was trying to manipulate an existing migration, but I'm pretty sure we came to the same conclusion: that the change does not change the structure or value of anything referenced in the codebase (the skills table's structure has been changed since the aforementioned migration, including changing the name column from a string to json type). Do let me know if this still poses a problem though, thanks.

Reply:
After further investigation this seems fine.